app/controllers/criteria_controller.rb

```diff
@@ -6,6 +6,27 @@ def new
     @criterion = @criterion_type.constantize.new
   end
 
+  def create
+    @assignment = Assignment.find(params[:assignment_id])
+    @criterion_type = params[:criterion_type]
+    @criteria = @assignment.get_criteria
+    @criterion = @criterion_type.constantize.new
+    @criterion.assignment = @assignment
+    @criterion.max_mark = @criterion_type.constantize::DEFAULT_MAX_MARK
+    @criterion.position = @assignment.next_criterion_position
+    rubric = @criterion_type == 'RubricCriterion'
+    @criterion.set_default_levels if rubric
+    unless rubric ?
```

Comment:
Don't use the ternary operator across multiple lines, and certainly not as a condition in an `unless`.